lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -12,25 +11,20 @@ import 'package:fluffychat/utils/string_extension.dart';
 import 'package:fluffychat/utils/twake_snackbar.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:future_loading_dialog/future_loading_dialog.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
 
 import 'package:matrix/matrix.dart';
 
 import 'matrix_file_extension.dart';
 
 extension LocalizedBody on Event {
-  Future<LoadingDialogResult<MatrixFile?>> getFile(
-    BuildContext context,
-  ) async =>
-      TwakeDialog.showFutureLoadingDialogFullScreen(
-        future: downloadAndDecryptAttachment,
-      );
 
   Future<String?> saveFile(BuildContext context) async {
-    final matrixFile = await getFile(context);
+    TwakeSnackBar.show(context, L10n.of(context)!.downloadStarted);
+    
+    final matrixFile = await downloadAndDecryptAttachment();
 
-    return await matrixFile.result?.downloadFile(context);
+    return await matrixFile.downloadFile(context);
```

Comment:
please test with other kind of file: .zip, .apk, ...

Reply:
Can't upload large files (more than 70MB). I created an issue about that: https://github.com/linagora/twake-on-matrix/issues/1536 .

I'll try with lighter files

Reply:
[Capture vidéo du 05-03-2024 13:41:38.webm](https://github.com/linagora/twake-on-matrix/assets/31937920/d0094a95-f748-42dd-9f68-9bbc8a2701c0)

Reply:
I run it but can not see download action for `zip` file
<img width="596" alt="Screenshot 2024-03-06 at 16 14 01" src="https://github.com/linagora/twake-on-matrix/assets/6462404/62d2f049-1307-411f-bbf5-df6d87aad6d1">